src/components/CategoryFilter.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import styled from 'styled-components';
+import { useSelector } from 'react-redux';
+import PropTypes from 'prop-types';
+
+const CategoryFilterSelect = styled.select`
+  background: lightblue;
+  color: white;
+`;
+
+export default function CategoryFilter(props) {
+  let count = 0;
+
+  const { handleChange } = props;
+  const categories = useSelector(state => state.filter.categories);
+
+  return (
+    <CategoryFilterSelect onChange={handleChange}>
+      <option value="All">All</option>
+      {categories.map(category => {
+        count += 1;
+        return <option value={category} key={count}>{category}</option>;
+      })}
+      <option>Horror</option>
```

Comment:
why do you have this here?  
The only category that shouldn't be in the categories list is `all` see [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/milestone4/AllReturnsAllCategories.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/44896829/)